dawn/src/dawn/Unittest/IIRBuilder.h

```diff
@@ -125,8 +125,20 @@ class IIRBuilder {
     DAWN_ASSERT(si_);
     auto ret = std::make_unique<iir::DoMethod>(iir::Interval(s, e), si_->getMetaData());
     ret->setID(si_->nextUID());
-    [[maybe_unused]] int x[] = {
-        (DAWN_ASSERT(stmts), ret->getAST().push_back(std::move(stmts)), 0)...};
+    int x[] = {(DAWN_ASSERT(stmts), ret->getAST().push_back(std::move(stmts)), 0)...};
+    computeAccesses(si_.get(), ret->getAST().getStatements());
+    ret->updateLevel();
+    return ret;
+  }
+
+  template <typename... Stmts>
+  std::unique_ptr<iir::DoMethod> vregion(sir::Interval::LevelKind s, sir::Interval::LevelKind e,
+                                         int offsetLow, int offsetHigh, Stmts&&... stmts) {
+    DAWN_ASSERT(si_);
+    auto ret = std::make_unique<iir::DoMethod>(iir::Interval(s, e, offsetLow, offsetHigh),
+                                               si_->getMetaData());
+    ret->setID(si_->nextUID());
+    int x[] = {(DAWN_ASSERT(stmts), ret->getAST().push_back(std::move(stmts)), 0)...};
```

Comment:
this will give a warning about `x` being unused. There are several options:
- `void((int[]){...})`
- `int x[] = {...}; (void) x;`
- `[[maybe_unused]] int x[] = {...}`